client/ui/qml/Pages2/PageSettingsSplitTunneling.qml

```diff
@@ -179,124 +169,120 @@ PageType {
                     }
                 }
             }
-
-            KeyNavigation.tab: {
-                return sites.count > 0 ?
-                            sites :
-                            searchField.textField
-            }
         }
     }
 
-    FlickableType {
-        id: fl
+    ListView {
+        id: sites
+
         anchors.top: header.bottom
         anchors.topMargin: 16
-        contentHeight: col.implicitHeight + addSiteButton.implicitHeight + addSiteButton.anchors.bottomMargin + addSiteButton.anchors.topMargin
+        width: parent.width
+
+        height: 200 // TODO: Change to correct height
 
         enabled: root.pageEnabled
 
-        Column {
-            id: col
-            anchors.top: parent.top
-            anchors.left: parent.left
-            anchors.right: parent.right
+        property bool isFocusable: true
 
```

Comment:
```suggestion

        ScrollBar.vertical: ScrollBar {}
```

Reply:
The default behavior of the ScrollBar is not working correctly. To fix this, you can initialize the ScrollBar as an empty object. Qt will then set the necessary default values automatically.

Reply:
I added ScrollBarType component and use it on all ListViews